src/main/java/io/inscopemetrics/kairosdb/aggregators/HistogramCountAggregator.java

```diff
@@ -41,16 +40,15 @@
      * Public constructor.
      *
      * @param dataPointFactory A factory for creating DoubleDataPoints
-     * @throws KairosDBException on error
      */
     @Inject
-    public HistogramCountAggregator(final DoubleDataPointFactory dataPointFactory) throws KairosDBException {
+    public HistogramCountAggregator(final DoubleDataPointFactory dataPointFactory) {
         this.dataPointFactory = dataPointFactory;
     }
 
     @Override
     protected RangeSubAggregator getSubAggregator() {
-        return new HistogramMeanDataPointAggregator();
+        return new HistogramCountDataPointAggregator();
```

Comment:
huh - was this just straight-up broken before? (Mean -> Count seems like a functional change)

Reply:
Not broken; the inner class was just named wrong. Inner class names are scoped within their containers in Java, so it was using the correct class, it was just a very misleading name.